src/pages/restaurant/components/menu/components/cart/Cart.js

```diff
@@ -0,0 +1,91 @@
+//Libraries
+import React from 'react';
+import PropTypes from 'prop-types';
+
+//Helpers
+import _noop from 'lodash/noop';
+import { getTotalAmount } from './helpers/cart.helper';
+import { getSubHeading, getTotalItems } from '../../helpers/menu.helper';
+
+//Icons
+import rupee from '../../../../../../images/rupee.svg';
+
+//Components
+import CartItemsList from './components/cartItemsList';
+import IconWithLabel from '../../../../../../commonComponents/iconWithLabel';
+import Button from '../../../../../../commonComponents/button';
+import { BUTTON_TYPES } from '../../../../../../commonComponents/button';
+
+//CSS
+
+import './cart.css';
+
+function Cart(props) {
+  const {
+    dishes,
+    quantityPerDishById,
+    onCheckout,
+    incrementDishQuantity,
+    decrementDishQuantity,
+  } = props;
+  const totalAmount = getTotalAmount(dishes, quantityPerDishById);
+  const totalItems = getTotalItems(quantityPerDishById);
+  return (
+    <div className='cart'>
+      <div className='cartItems'>
+        <div>
+          <h1>Cart</h1>
+          <p className='lightWeightText'>{getSubHeading(totalItems, 'Item')}</p>
+        </div>
+        <CartItemsList
+          dishes={dishes}
+          quantityPerDishById={quantityPerDishById}
+          incrementDishQuantity={incrementDishQuantity}
```

Comment:
incrementDishQuantity doesn't look like a function name, change it based on it's functionality e.g onIncrementClick